### PR TITLE
fic: certification links now open in a new tab

### LINKED ISSUE #39127: Audit navigation (opening new tabs only when necessary)
You can go to [my profile](https://www.freecodecamp.org/mot01) and click on one of the certifications - it opens in the current tab. The only way to go back is to use the browsers back button. Seems like those should open in a new tab.

### PR DESCRIPTION
**Pull request by DavidJSykes**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39127

### CODE REVIEW

**Comment by DavidJSykes** [comment]
@raisedadead I'm sorry if this PR isn't correctly done, I originally assigned it to my initial PR and not the issue

**Comment by raisedadead** [comment]
Nice work @DavidJSykes. 

I had a quick glance and this is what we want, someone will do a review and get back to you if needed.

Thanks.

**Review by huyenltnguyen — DISMISSED** [review]
@DavidJSykes Your PR looks good to me :tada: 

I just have a small nit regarding the prop `sameTab` passed to `Link`, but I'll let the core team decide.

Thank you for your contribution!

File: client/src/components/profile/components/Certifications.js
Comment (raisedadead) [reply client/src/components/profile/components/Certifications.js:47]: Yep, I agree. @DavidJSykes we should be able to leave out the `sameTab` prop

Comment (DavidJSykes) [reply client/src/components/profile/components/Certifications.js]: Did I update that correctly?

Comment (huyenltnguyen) [reply client/src/components/profile/components/Certifications.js]: You did! Thanks for the update. I'm resolving the discussion now.

**Review by huyenltnguyen — APPROVED** [review]
LGTM. @DavidJSykes Thanks a lot for your contribution 🎉

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks @DavidJSykes, for this pull request and contributing to the code-base for the first time.
We will be looking forward for more contributions in future.

Cheers and happy contributing!

PS: Merging this PR will soil your fork. Please go thru our contributing guide on how to resolve this.

**Pull request merged**
